docs/standard/exceptions.md

```diff
@@ -0,0 +1,528 @@
+---
+title: Handling and throwing exceptions in .NET
+description: Understand how to use exceptions in .NET
+keywords: .NET, .NET Core
+author: mairaw
+manager: wpickett
+ms.date: 06/20/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: bf116df6-0042-46bf-be13-b69864816210
+---
+
+# Handling and throwing exceptions in .NET
+
+Applications must be able to handle errors that occur during execution in a consistent manner. .NET provides a model for notifying applications of errors in a uniform way: .NET operations indicate failure by throwing exceptions.
+
+## Exceptions
+
+An exception is any error condition or unexpected behavior that is encountered by an executing program. Exceptions can be thrown because of a fault in your code or in code that you call (such as a shared library), unavailable operating system resources, unexpected conditions that the runtime encounters (such as code that cannot be verified), and so on. Your application can recover from some of these conditions, but not from others. Although you can recover from most application exceptions, you cannot recover from most runtime exceptions.
+
+In .NET, an exception is an object that inherits from the [System.Exception](https://docs.microsoft.com/dotnet/core/api/System.Exception) class. An exception is thrown from an area of code where a problem has occurred. The exception is passed up the stack until the application handles it or the program terminates.
+
+## Exceptions vs. traditional error-handling methods
+
+Traditionally, a language's error-handling model relied on either the language's unique way of detecting errors and locating handlers for them, or on the error-handling mechanism provided by the operating system. .NET implements exception handling with the following features:
+
+- Handles exceptions without regard for the language that generates the exception or the language that handles the exception.
+
+- Does not require any particular language syntax for handling exceptions, but allows each language to define its own syntax.
+
+- Allows exceptions to be thrown across process and even machine boundaries.
```

Comment:
True, but it feels like this is a bit of an overreach. Does ".NET allow this" or is this enabled by virtue of .NET serialization?